Carthage/Build/Mac/CryptoSwift.framework/Versions/A/Headers/CryptoSwift.h

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
These are Carthage build artefacts which should not be committed to the repository.